pkgs/top-level/python-packages.nix

```diff
@@ -2640,6 +2640,8 @@ in {
 
   geojson-client = callPackage ../development/python-modules/geojson-client { };
 
+  geomag = callPackage ../development/python-modules/geomag { };
```

Comment:
You appear to be putting a fork (albeit a more actively-developed one) into the global python package namespace. This could be confusing for anyone who expects geomag to be the upstream version which appears in pypi, or awkward if somebody wants to package something in the future which depends on upstream geomag.

Not saying we _can't_ have rinigus' geomag as pythonPackages.geomag, but it's worth thinking through carefully.

Reply:
Maybe we should encourage rinigus to publish it on PyPI.
It seems like the data bundled with the package is outdated in older versions and that's the only thing this fork updates. Hence it doesn't seem problematic.

Reply:
Yeah. The original 'upstream' hasn't been updated since 2014, so I'm inclined to consider it abandoned / unsupported, with the fork and the fork fork appearing to be maintained and updated versions of it, and thus a reasonable candidate to package as simply 'geomag' (In situations where upstream is still going and the fork has just decided to go in a different direction, I would be more opposed to the fork being the main package in nixpkgs).

Reply:
@rinigus How do you feel about publishing your fork of geomag on PyPI?

Reply:
Not thrilled about it. As Pure Maps has been rewritten using C++ main(), I may switch to some other library for that info. Not the highest priority, but would prefer that than to start packaging someones outdated code which probably has functionality that I don't use. And, as a result, may have bugs that I am not hitting.

Reply:
Understandable. Thanks for your great work anyway!